sw/airborne/subsystems/navigation/common_flight_plan.h

```diff
@@ -55,6 +55,7 @@ void nav_goto_block(uint8_t block_id);
 #define Label(x) label_ ## x:
 #define Goto(x) { goto label_ ## x; }
 #define Return() { nav_block=last_block; nav_stage=last_stage; block_time=0;}
+#define ReturnToStartPrevBlock() { nav_block=last_block; nav_stage=0; block_time=0;}
```

Comment:
The `Return` instruction can be used from a special flight plan instruction. Shall we do the same for this, or with an attribute of the existing instruction ?